src/lib.rs

```diff
@@ -22,7 +22,7 @@
 //! | Dragonfly BSD     | `*‑dragonfly`      | [`getrandom()`][22] if available, otherwise [`/dev/random`][8]
 //! | Solaris, illumos  | `*‑solaris`, `*‑illumos` | [`getrandom()`][9] if available, otherwise [`/dev/random`][10]
 //! | Fuchsia OS        | `*‑fuchsia`        | [`cprng_draw`][11]
-//! | Redox             | `*‑redox`          | [`rand:`][12]
+//! | Redox             | `*‑redox`          | [`/dev/urandom`][12]
```

Comment:
We may need to update the documentation link here

Reply:
The link simply points to relevant Redox source code. It would be nice to replace it with a proper documentation link, but AFAIK it currently does not exist.

Reply:
@jackpot51 does Redox have any docs describing either `rand:` or `/dev/urandom`?

Reply:
@josephlr not really